vroom/system.vroom

```diff
@@ -241,6 +241,12 @@ This function converts non-zero status codes into ShellError exceptions:
   $ 1 (status)
   ~ ERROR(ShellError): Error running: nothing '!@#$'
 
+  :let g:failcmd = '(echo "Some stderr output !@#$%s^&*()[]" >&2 && exit 1)'
```

Comment:
Possibly add a reference to #224 here? The important part is that the stderr contains a format string (i.e. `%s`).

Reply:
Done.